README.md

```diff
@@ -7,7 +7,7 @@
 
 Lagom is a Swedish word meaning *just right, sufficient*. Microservices are about creating services that are just the right size, that is, they have just the right level of functionality and isolation to be able to adequately implement a scalable and resilient system.
 
-Lagom focuses on ensuring that your application realises the full potential of the [Reactive Manifesto](http://reactivemanifesto.org), while delivering a high productivity development environment, and seamless production deployment experience.
+Lagom focuses on ensuring that your application realises the full potential of the [Reactive Manifesto](http://reactivemanifesto.org) while delivering a high productivity development environment, and seamless production deployment experience.
```

Comment:
```suggestion
Lagom focuses on ensuring that your application realizes the full potential of the [Reactive Manifesto](http://reactivemanifesto.org) while delivering a high productivity development environment, and seamless production deployment experience.
```
😄

Reply:
@abhiknoldur do you want to accept this suggestion?

Reply:
@ignasi35 Of course. Because it's not my suggestion, it's suggested Grammarly 😂

Reply:
@ignasi35 I think its depends on the territory in which the reader lives in because the word **realise** is meant for the British English and **realize** meant for American English which is also followed in Grammarly as well. Its totally upto you, I don't want to go with any specific territory that is why I'll suggest you can ignore that.

Reply:
We have authors in different regions, so that explains the inconsistency, but the official policy is to standardize on US style spelling (not standardise 😉).

Reply:
so that settles it.

@Abhiknoldur, as the creator of the PR, only you can accept the suggestion. Alternatively, I could do the change locally and push on your branch (but I'd rather keep you and @ihostage as authors of the change :-) ).

Reply:
~as the creator of the PR, only you can accept the suggestion.~

_As the owner of the branch and because I may not have permission to push into your branch_, only you can accept the suggestion.